### DISCUSSION TITLE
Discussion Session (Hacktoberfest 2023)

### DISCUSSION THREAD

**Discussion by lmas3009** [lmas3009/design2code2live#220] [General]
Hello, everyone,

How has your experience been while contributing to Design2Code2Live?

Contibutors: @Pariharx7 @zainbinfurqan @Atrika10 @harshil-junakiya @NereidaRondon @NyanCyanide @Aka-suvam @Ryan-Maddock-96 @SamuelWoniowei @saffi-786 @Devesh1509 @05hiteshbansal @avyuktsoni0731 @stktyagi @Rohit-Sharmaa

**Comment by Rohit-Sharmaa** [comment]
it has been very nice experience while contributing to Design2Code2Live. 

Thank you @lmas3009
